projects/ng-bulma-lib/src/lib/compenents/modal/bm-modal-card/modal-card-header.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { ModalRef } from '../modal-ref';
+
+@Component({
+  selector: 'bm-modal-card-header',
+  template: `
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        <ng-content></ng-content>
+      </p>
+      <button class="delete" aria-label="close" (click)="close('cross-click')"></button>
```

Comment:
class="delete" => class="close"

Reply:
class="delete" is default class of bulma and "close" class doesn't exist.

https://bulma.io/documentation/elements/delete/